vendor/DOtherSide/lib/CMakeLists.txt

```diff
@@ -24,15 +26,19 @@ macro(add_target name type)
         list(INSERT SOURCES 0 ${MM_FILES})
```

Comment:
See here, we are prepending on Mac

Reply:
Here it's workaround, in my new code it's not necessary.